### PR TITLE
Update amstelveen municipality to 2019

### PR DESCRIPTION
**Pull request by Charlottevm**
Some interesting changes:
- number of buildings has declined drastically
- decline of agricultural sector
- increase of number of inhabitants but decrease in number of houses

These changes seem to cause a significant decline in CO2 compared to the previous dataset.

### CODE REVIEW

File: datasets/GM0362_amstelveen/GM0362_amstelveen.derived.ad [hunk 1]
```diff
@@ -134,45 +134,45 @@
 - electric_vehicle_profile_5_share = 0.0
 - solar_pv_profile_1_share = 1.0
 - solar_pv_profile_2_share = 0.0
-- annual_infrastructure_cost_gas = 8.236905182997704
+- annual_infrastructure_cost_gas = 8.236905182997703e-06
```
Comment (noracato) [inline datasets/GM0362_amstelveen/GM0362_amstelveen.derived.ad:137]: This value has become a million times smaller. Is that correct?

File: datasets/GM0362_amstelveen/GM0362_amstelveen.derived.ad [hunk 2]
```diff
@@ -134,45 +134,45 @@
...
+- co2_emission_1990 = 0.346136633
 - coast_line = 0.0
 - interconnector_capacity = 60000.0
 - total_land_area = 41.21
-- number_of_buildings = 7009.0
-- number_of_cars = 38091.0
-- number_of_residences = 43045.0
-- number_of_inhabitants = 88602.0
-- offshore_suitable_for_wind = 258.769864280363
+- number_of_buildings = 1693.0
```
Comment (noracato) [inline datasets/GM0362_amstelveen/GM0362_amstelveen.derived.ad:143]: Another quick sanity check: 4500 buildings have dissapeared

File: datasets/GM0362_amstelveen/GM0362_amstelveen.derived.ad [hunk 3]
```diff
@@ -134,45 +134,45 @@
...
+- number_of_cars = 37955.0
+- number_of_residences = 42035.0
+- number_of_inhabitants = 90837.0
+- offshore_suitable_for_wind = 30060.0
```
Comment (noracato) [inline datasets/GM0362_amstelveen/GM0362_amstelveen.derived.ad:147]: Another big change. Amstelveen does not have large lakes nor seas, right?

File: datasets/GM0362_amstelveen/GM0362_amstelveen.derived.ad [hunk 4]
```diff
@@ -134,45 +134,45 @@
...
+- non_energetic_emissions_other_ghg_agriculture_manure = 0.001632877
+- non_energetic_emissions_other_ghg_agriculture_soil_cultivation = 0.004647203
+- non_energetic_emissions_other_ghg_agriculture_other = 0.0
+- non_energetic_emissions_other_ghg_waste_management = 0.001570105
+- indirect_emissions_co2 = 0.002223454
+- number_of_detached_houses = 1235.0220116757584
+- number_of_apartments = 21209.59218585511
+- number_of_semi_detached_houses = 869.9462867260025
+- number_of_corner_houses = 4805.321801129293
+- number_of_terraced_houses = 13915.11771461384
```
Comment (noracato) [inline datasets/GM0362_amstelveen/GM0362_amstelveen.derived.ad:173]: One more sanity check: There are less homes in 2019 than there were in 2016.

**Review by noracato — APPROVED** [review]
I went over the files quickly, and there were a few things that caught my eye. Maybe it's all correct and expected, but I marked them just to be sure 🙂 if you think it's all good, feel free to merge!

File: datasets/GM0362_amstelveen/GM0362_amstelveen.derived.ad (hunk 1 above)
Comment (Charlottevm) [reply datasets/GM0362_amstelveen/GM0362_amstelveen.derived.ad:137]: This was fixed during the NL2019 update

File: datasets/GM0362_amstelveen/GM0362_amstelveen.derived.ad (hunk 3 above)
Comment (Charlottevm) [reply datasets/GM0362_amstelveen/GM0362_amstelveen.derived.ad:147]: No :-) But this number has been set the same for all municipalities

File: datasets/GM0362_amstelveen/GM0362_amstelveen.derived.ad (hunk 4 above)
Comment (Charlottevm) [reply datasets/GM0362_amstelveen/GM0362_amstelveen.derived.ad:173]: Again it seems right with the 2019 data we have from CBS, could again be a mistake in the previous dataset

**Comment by Charlottevm** [comment]
> I went over the files quickly, and there were a few things that caught my eye. Maybe it's all correct and expected, but I marked them just to be sure 🙂 if you think it's all good, feel free to merge!

Thanks for the thorough check! :-)

**Pull request merged**